config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -0,0 +1,76 @@
.image.should.be.able.to.pull.image.from.gcr.io.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.running.a.container.with.a.new.image.should.not.be.able.to.pull.from.private.registry.without.secret.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.running.a.container.with.a.new.image.should.not.be.able.to.pull.image.from.invalid.registry.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.running.a.container.with.a.new.image.should.not.be.able.to.pull.non-existing.image.from.gcr.io.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.report.termination.message.as.empty.when.pod.succeeds.and.TerminationMessagePolicy.FallbackToLogOnError.is.set.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.report.termination.message.from.file.when.pod.succeeds.and.TerminationMessagePolicy.FallbackToLogOnError.is.set.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.report.termination.message.from.log.output.if.TerminationMessagePolicy.FallbackToLogOnError.is.set.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.report.termination.message.if.TerminationMessagePath.is.set.as.non-root.user.and.at.a.non-default.path.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.report.termination.message.if.TerminationMessagePath.is.set.\\[NodeConformance\\]|\\[k8s.io\\].Container.Runtime.blackbox.test.when.starting.a.container.that.exits.should.run.with.the.expected.status.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.When.creating.a.container.with.runAsUser.should.run.the.container.with.uid.0.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.When.creating.a.container.with.runAsUser.should.run.the.container.with.uid.65534.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.When.creating.a.pod.with.privileged.should.run.the.container.as.unprivileged.when.false.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.When.creating.a.pod.with.readOnlyRootFilesystem.should.run.the.container.with.readonly.rootfs.when.readOnlyRootFilesystem=true.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.When.creating.a.pod.with.readOnlyRootFilesystem.should.run.the.container.with.writable.rootfs.when.readOnlyRootFilesystem=false.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.when.creating.containers.with.AllowPrivilegeEscalation.should.allow.privilege.escalation.when.not.explicitly.set.and.uid.!=.0.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.when.creating.containers.with.AllowPrivilegeEscalation.should.allow.privilege.escalation.when.true.\\[NodeConformance\\]|\\[k8s.io\\].Security.Context.when.creating.containers.with.AllowPrivilegeEscalation.should.not.allow.privilege.escalation.when.false.\\[NodeConformance\\]"
```

Comment:
er why are we using this huge regex? :O (just curious)

Reply:
Not all tests will work on Windows. For example: tests that set securityContext, or tests that set mode on files and expects that to be the same ( *nix / Windows differs much here ) etc. Sadly, we can't just skip the whole group of tests, as some of them should work.

Reply:
is the list shorter if enable conformance then skip tests rather than listing the ones we will run?

Reply:
That;s what we have. --ginkgo.focus=Conformance|NodeConformance , --skip=huge regex of tests we skip.

Reply:
:man_facepalming: thanks, that's really unfortunate, hopefully the "profiles" get sorted out so we can use a simpler regex

Reply:
Can't wait for that :) If there's any help we can provide from the windows side, I'll be happy to lend a hand.

Reply:
this one is up to sig-architecture / the conformance work group. we can help, but ultimately adding a tag will be the easy part, deciding which should be tagged and how will require more discussion with those groups 😬